### DISCUSSION TITLE
VLANs and pfelk

### DISCUSSION THREAD

**Discussion by stevefxp1** [pfelk/pfelk#369] [Q&A] [answered]
When changing the interfaces configuration file how do I call out vlans as part of this configuation?? Do I just call the interface as it shows in OPNsense, ie igb0_vlan12?

**Comment by a3ilson** [comment, marked as answer]
@stevefxp1 - you'll simply need to adjust the [20-interfaces.conf](https://github.com/pfelk/pfelk/blob/main/etc/pfelk/conf.d/20-interfaces.conf)

Line 40-46 contains an example of a vlan:
```
    ### Change interface as desired ###
    if [interface][name] =~ /^igb1_vlan2000$/ {
      mutate {
        add_field => { "[interface][alias]" => "VLAN" }
        add_field => { "[network][name]" => "Isolated Network" }
      }
    }

```
